robot_smach_states/src/robot_smach_states/navigation/navigation.py

```diff
@@ -40,14 +40,16 @@ def execute(self, userdata=None):
 
 
 class getPlan(smach.State):
-    def __init__(self, robot, constraint_function, speak=True):
+    def __init__(self, robot, constraint_function, input_keys=[], output_keys=[], speak=True):
```

Comment:
Lists as default args are not allowed, see https://docs.python-guide.org/writing/gotchas/